src/niclips/io.py

```diff
@@ -13,6 +14,27 @@
 from niclips.image._convert import topil
 from niclips.typing import StrPath
 
+try:
+    import nifti
+
+    HAVE_NIFTI = True
+except ImportError:  # pragma: no cover
+    HAVE_NIFTI = False
+
+
+def load_nifti(fpath: str | Path) -> nib.Nifti1Image:
```

Comment:
Should this have a `use_niftilib` flag or `backend` arg or sth? This way we can select which io backend to use.

Reply:
Yep, I am open to that - do we also want to expose this to the CLI?

Reply:
Added a use flag in 3a4d40e. I think a `backend` arg might be more than needed at the moment unless we plan on adding additional packages / libraries.